build-tools/src/main/java/co/elastic/apm/compile/tools/publishing/tasks/PostDeployTask.java

```diff
@@ -47,16 +47,21 @@ public void execute() {
         updateNextVersion(gradlePropertiesFile, properties, newVersion);
         updateChangelog(currentVersion);
 
-        publishChanges();
+        createPullRequestWithChanges(newVersion);
```

Comment:
Probably we should use `currentVersion` instead? Since it's the one that was just released by the time this task runs.

Reply:
```suggestion
        createPullRequestWithChanges(currentVersion);
```